apps/website/src/content/docs/progressindicator.mdx

```diff
@@ -1,29 +1,90 @@
 ---
 title: Progress indicator
-description: Progress indicators express an unspecified wait time or display the length of a process.
+description: Progress indicators express the length of in-progress operations.
 thumbnail: ProgressIndicator
 ---
 
 <p>{frontmatter.description}</p>
 
-## Progress linear
+## Usage
 
-<Placeholder componentPageName='progresslinear--determinate' />
+There are two types of progress indicators which are linear and radial. Linear indicators show progress on a linear bar, while radial indicators display progress in a circular form.
```

Comment:
We should use the exact component names here, and maybe use a list for readability.

```suggestion
There are two types of progress indicators available as separate components:

- `ProgressLinear`: shows progress on a linear bar
- `ProgressRadial`: shows progress in a circular form.
```

Reply:
This comment is actually applicable throughout the page. Most of the references to "linear indicator" and "radial indicator" could be replaced with the component names.

Reply:
I have explicitly mentioned these components.